inc/Smartling/Base/SmartlingCoreUploadTrait.php

```diff
@@ -88,6 +88,9 @@ private function setFileUriIfNullId(SubmissionEntity $submission): SubmissionEnt
     private function createTargetContent(SubmissionEntity $submission): SubmissionEntity
     {
         $submission = $this->getFunctionProxyHelper()->apply_filters(ExportedAPI::FILTER_SMARTLING_PREPARE_TARGET_CONTENT, $submission);
+        if (!$submission instanceof SubmissionEntity) {
```

Comment:
Maybe make it more explicit by adding brackets. I will not be surprised if PHP will invert value of `$submission` and then will check its type
```suggestion
        if (!($submission instanceof SubmissionEntity)) {
```

Reply:
! has lower precedence than instanceof, but explicit is better